login-workflow/src/new-architecture/contexts/RegistrationContext/RegistrationContextProvider.test.tsx

```diff
@@ -0,0 +1,145 @@
+import React from 'react';
+import { render, cleanup, screen } from '@testing-library/react';
+import '@testing-library/jest-dom';
+import { RegistrationContextProvider } from './provider';
+import i18next from 'i18next';
+import { dictionaries } from './dictionaries';
+import { useRegistrationContext } from './context';
+import { RegistrationContextProviderProps } from './types';
+import { renderHook } from '@testing-library/react-hooks';
+
+afterEach(cleanup);
+
+const i18nRegistrationInstance = i18next.createInstance(
+    {
+        lng: 'en',
+        fallbackLng: 'en',
+        ns: ['bluiRegistration'],
+        defaultNS: 'bluiRegistration',
+        load: 'languageOnly',
+        detection: {
+            order: ['querystring', 'localStorage', 'navigator'],
+            caches: ['localStorage'],
+            lookupLocalStorage: 'blui-registration-i18nextLng',
+        },
+        react: { useSuspense: false },
+        interpolation: { escapeValue: false },
+        resources: {
+            en: {
+                bluiRegistration: {
+                    ...dictionaries.english.translation,
+                },
+            },
+            fr: {
+                bluiRegistration: {
+                    ...dictionaries.french.translation,
+                },
+            },
+            es: {
+                bluiRegistration: {
+                    ...dictionaries.spanish.translation,
+                },
+            },
+            zh: {
+                bluiRegistration: {
+                    ...dictionaries.chinese.translation,
+                },
+            },
+            pt: {
+                bluiRegistration: {
+                    ...dictionaries.portuguese.translation,
+                },
+            },
+        },
+    },
+    // We must provide a function as second parameter, otherwise i18next errors
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    (err, _t) => {
+        // @TODO: Handle this error state more graciously
+        // eslint-disable-next-line no-console
+        if (err) return console.log(err);
+    }
+);
+
+describe('RegistrationContextProvider', () => {
+    const defaultProps = {
+        language: 'en',
+        i18n: i18nRegistrationInstance,
+        navigate: (): void => {},
+        routeConfig: {},
+    };
```

Comment:
Instead of writing the separate instace, you can reuse the old defined.

const defaultProps = {
        language: 'en',
        i18nRegistrationInstance,
        navigate: (): void => {},
        routeConfig: {},
    };